packages/react-dom/server.node.js

```diff
@@ -9,4 +9,11 @@
 
 'use strict';
 
-module.exports = require('./src/server/ReactDOMServerNode');
+var ReactDOMServer = require('./src/server/ReactDOMServerNode');
+
+// TODO: decide on the top-level export form.
+// This is hacky but makes it work with both Rollup and Jest
+// Note: when changing this, also consider https://github.com/facebook/react/issues/11526
```

Comment:
Actually let's move this note to `export default {` (in both cases).
Because right now it would be possible to remove `default` again and tests/build would still pass.

Reply:
sure, that makes sense. updated